config/sites/developer/config.yaml

```diff
@@ -0,0 +1,18 @@
+base: developer/
+baseVariants: {  }
+errorHandling: {  }
+languages:
+  -
+    title: English
+    enabled: true
+    languageId: '0'
+    base: /
+    typo3Language: default
+    locale: en_US.UTF-8
+    iso-639-1: en
+    navigationTitle: English
+    hreflang: en-us
+    direction: ltr
+    flag: global
```

Comment:
Please also let's have additional languages (either now or in a separate ticket).

Reply:
Created issue #4